cmd/notation/plugin/list.go

```
@@ -68,7 +71,27 @@ func listPlugins(command *cobra.Command) error {
 			}
 		}
 		fmt.Fprintf(tw, "%s\t%s\t%s\t%v\t%v\t\n",
-			n, metaData.Description, metaData.Version, metaData.Capabilities, err)
+			n, metaData.Description, metaData.Version, metaData.Capabilities, userFriendlyError(err))
 	}
 	return tw.Flush()
 }
+
+// userFriendlyError optimizes the error message for the user.
+func userFriendlyError(err error) error {
+	if err == nil {
+		return nil
+	}
+	var pathError *fs.PathError
+	if errors.As(err, &pathError) {
+		// for plugin does not exist
+		if errors.Is(pathError, fs.ErrNotExist) {
+			return fmt.Errorf("%w. Each plugin executable must be placed in the $PLUGIN_DIRECTORY/{plugin-name} directory, with the executable named as 'notation-{plugin-name}'", pathError)
```

Do we need to return the error, because from your example shown in the PR description, the error itself is verbose and provides very little information.
In addition, can we mention `notation plugin install` command in the error msg as an actionable next step for users?

---

Updated error message.